lib/ddtrace/tracer.rb

```diff
@@ -161,6 +161,13 @@ def active_span
       @buffer.get()
     end
 
+    # stats returns a dictionary of stats about the writer.
```

Comment:
do we need this stat? We already have stats for the ``Writer`` and the ``Transport``. Can't we put this stat (number of spans for the current trace) when the trace is completed (so in the ``Writer``)? Just proposing because probably I'm missing something :)